ci-operator/step-registry/ipi/conf/gcp/secureboot/ipi-conf-gcp-secureboot-commands.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+PATCH="${SHARED_DIR}/secure_boot.yaml.patch"
+cat > "${PATCH}" << EOF
+controlPlane:
+  platform:
+    gcp:
+      secureBoot: "${SECURE_BOOT}"
```

Comment:
.yml should document supported env vars used in this script

Reply:
Also, do we need it parametrized? iiuc the install config section just needs `Enabled` set any time we include this step

Reply:
True, I just copy pasted it from the previous config when it was optional, no need for env var now.